graylog2-web-interface/src/views/components/Field.tsx

```diff
@@ -24,14 +24,14 @@ import CustomPropTypes from './CustomPropTypes';
 import FieldActions from './actions/FieldActions';
 import InteractiveContext from './contexts/InteractiveContext';
 
-type Props = {|
-  children?: React.Node,
+type Props = {
+  children?: React.ReactNode,
   disabled?: boolean,
   name: string,
-  menuContainer: ?HTMLElement,
+  menuContainer?: HTMLElement,
```

Comment:
Even though the prop is optional, shouldn't this be `HTMLElement | null | undefined`?